wrapper/src/main/java/software/amazon/jdbc/plugin/failover/ClusterAwareReaderFailoverHandler.java

```diff
@@ -72,9 +74,11 @@ public class ClusterAwareReaderFailoverHandler implements ReaderFailoverHandler
    */
   public ClusterAwareReaderFailoverHandler(
       PluginService pluginService,
+      ExceptionManagerService exceptionManagerService,
```

Comment:
Refer to my previous comment. It's better to extend PluginService with ExceptionManagerService .